src/docs/asciidoc/udp-server.adoc

```diff
@@ -326,3 +326,70 @@ public class Application {
 }
 ----
 ====
+
+== Metrics
+UDP server supports built-in integration with https://micrometer.io/[`Micrometer`].
```

Comment:
Change it to be: The UDP server....

The rule is that the addition of another word after the name forces the inclusion of an article. In this case, "server" after "UDP" forces the addition of "The." If you said, "UDP supports...," that would be fine (but not what you want to say, since "server" is a key bit of information in the sentence. 

I hope you don't mind an explanation. I try to explain when I see a pattern. (I used to be a teacher.)